saleor/graphql/webhook/subscription_types.py

```diff
@@ -0,0 +1,407 @@
+        return checkout
+
+
+class CheckoutCreated(ObjectType, CheckoutBase):
+    ...
+
+
+class CheckoutUpdated(ObjectType, CheckoutBase):
+    ...
+
+
+class PageBase(AbstractType):
+    page = graphene.Field(
+        "saleor.graphql.page.types.Page",
+        description="Look up a page.",
+    )
+
+    @staticmethod
+    def resolve_page(root, _info):
+        _, page = root
+        return page
+
+
+class PageCreated(ObjectType, PageBase):
+    ...
+
+
+class PageUpdated(ObjectType, PageBase):
+    ...
+
+
+class PageDeleted(ObjectType, PageBase):
+    ...
+
+
+class Event(Union):
+    class Meta:
+        types = (
+            OrderCreated,
+            OrderUpdated,
+            OrderConfirmed,
+            OrderFullyPaid,
+            OrderCancelled,
+            OrderFulfilled,
+            DraftOrderCreated,
+            DraftOrderUpdated,
+            DraftOrderDeleted,
+            ProductCreated,
+            ProductUpdated,
+            ProductDeleted,
+            ProductVariantCreated,
+            ProductVariantUpdated,
+            ProductVariantOutOfStock,
+            ProductVariantBackInStock,
+            ProductVariantDeleted,
+            SaleCreated,
+            SaleUpdated,
+            SaleDeleted,
+            InvoiceRequested,
+            InvoiceDeleted,
+            InvoiceSent,
+            FulfillmentCreated,
+            FulfillmentCanceled,
+            CustomerCreated,
+            CustomerUpdated,
+            CollectionCreated,
+            CollectionUpdated,
+            CollectionDeleted,
+            CheckoutCreated,
+            CheckoutUpdated,
+            PageCreated,
+            PageUpdated,
+            PageDeleted,
+        )
+
+    @classmethod
+    def get_type(cls, object_type: str):
+        types = {
+            WebhookEventAsyncType.ORDER_CREATED: OrderCreated,
+            WebhookEventAsyncType.ORDER_UPDATED: OrderUpdated,
+            WebhookEventAsyncType.ORDER_CONFIRMED: OrderConfirmed,
+            WebhookEventAsyncType.ORDER_FULLY_PAID: OrderFullyPaid,
+            WebhookEventAsyncType.ORDER_FULFILLED: OrderFulfilled,
+            WebhookEventAsyncType.ORDER_CANCELLED: OrderCancelled,
+            WebhookEventAsyncType.DRAFT_ORDER_CREATED: DraftOrderCreated,
+            WebhookEventAsyncType.DRAFT_ORDER_UPDATED: DraftOrderUpdated,
+            WebhookEventAsyncType.DRAFT_ORDER_DELETED: DraftOrderDeleted,
+            WebhookEventAsyncType.PRODUCT_CREATED: ProductCreated,
+            WebhookEventAsyncType.PRODUCT_UPDATED: ProductUpdated,
+            WebhookEventAsyncType.PRODUCT_DELETED: ProductDeleted,
+            WebhookEventAsyncType.PRODUCT_VARIANT_CREATED: ProductVariantCreated,
+            WebhookEventAsyncType.PRODUCT_VARIANT_UPDATED: ProductVariantUpdated,
+            WebhookEventAsyncType.PRODUCT_VARIANT_OUT_OF_STOCK: ProductVariantOutOfStock,  # noqa
+            WebhookEventAsyncType.PRODUCT_VARIANT_BACK_IN_STOCK: ProductVariantBackInStock,  # noqa
```

Comment:
IMO we shouldn't use `noqa` to mute line length errors.
@maarcingebala what do you think?

Reply:
Why? There is no easy way to format this particular line.

Reply:
Actually there is. @fowczarek suggested that putting key in parenthesis will let us put it in new line. I tried and it works.

Reply:
👍